addons/survey/static/tests/tours/survey_chained_conditional_questions.js

```diff
@@ -10,25 +10,45 @@ tour.register('test_survey_chained_conditional_questions', {
     {
         content: 'Click on Start',
         trigger: 'button.btn:contains("Start")',
+    }, {
+        content: 'Answer Q1 with Answer 2',
+        trigger: 'div.js_question-wrapper:contains("Q1") label:contains("Answer 2")',
+    }, {
+        content: 'Check that Q4 and Q5 are visible',
+        trigger: 'div.js_question-wrapper:contains(Q4)',
+        extra_trigger: 'div.js_question-wrapper:contains(Q5)',
+        run: () => {
+            const selector = 'div.js_question-wrapper.d-none';
+            if (document.querySelectorAll(selector).length !== 2) {
+                throw new Error('Q2 and Q3 should be hidden.');
+            }
+        },
     }, {
         content: 'Answer Q1 with Answer 1',
         trigger: 'div.js_question-wrapper:contains("Q1") label:contains("Answer 1")',
     }, {
         content: 'Answer Q2 with Answer 1',
         trigger: 'div.js_question-wrapper:contains("Q2") label:contains("Answer 1")',
+        run: function(actions) {
```

Comment:
Can we make this an arrow function like the others?

Reply:
All right, let's keep the little extra complexity so we avoid one extra step. 

Could you please just add the missing ` `  after `function`, and add a comment next to the click to show that this should "trigger" Q3, so it's more obvious to the reader as we've just checked above that it should be hidden?